stdlib/list.mli

```diff
@@ -201,18 +201,18 @@ val concat_map : ('a -> 'b list) -> 'a list -> 'b list
 *)
 
 val fold_left_map :
-  ('a -> 'b -> 'a * 'c) -> 'a -> 'b list -> 'a * 'c list
+  ('acc -> 'b -> 'acc * 'c) -> 'acc -> 'b list -> 'acc * 'c list
```

Comment:
Where is the type `'a`?

Reply:
'a is no more there because it is replaced by 'acc; which means accumulator

Reply:
Using such non-contextual replacement leaves a gap in the type variable names.

Here, the type of the element of the first list appearing in the list is `'b`,in other words the second type variable name by default,  and the type of the second list of `'c`, which is the third type variable by default.

This mismatch with the usual naming scheme is potentially confusing for users and should be fixed by using the default type variable names outside of the accumulator type.

Reply:
Ok, so you want the first unamed type variable to be 'a, the second 'b, etc.
I can do that.
I'll update the PR.